build.gradle

```diff
@@ -5,24 +5,18 @@ buildscript {
     maven { url "https://plugins.gradle.org/m2/" }
   }
   dependencies {
-    classpath "com.netflix.nebula:nebula-kotlin-plugin:$kotlinVersion"
-    classpath "com.netflix.spinnaker.gradle:spinnaker-gradle-project:4.0.0"
-    classpath "org.jetbrains.dokka:dokka-gradle-plugin:0.9.15"
-    classpath "org.jetbrains.kotlin:kotlin-allopen:$kotlinVersion"
-  }
-  configurations.all {
-    resolutionStrategy {
-      eachDependency { details ->
-        if (details.requested.group == "org.jetbrains.kotlin") {
-          details.useVersion kotlinVersion
```

Comment:
No longer need this as we get `kotlin-bom` transitively from `kork-bom` so that aligns everything.